documentation/index.html.md

```diff
@@ -22,19 +22,24 @@ If you are comfortable with JavaScript, the epicenter.js library is an easy way
 <a name="concepts"></a>
 ###Concepts in Epicenter.js
 
-The epicenter.js library is a set of services (adapters) and managers to help streamline your work with the underlying Epicenter APIs.
+The epicenter.js library is a set of services (adapters) and managers to help streamline your work with the underlying Epicenter APIs. Typically, you'll develop using the managers:
```

Comment:
Let's also briefly describe what the services and managers do:

Services encapsulate calls to the Epicenter REST APIs. Managers are responsible for configuring, sequencing, and synchronizing services to perform common application tasks. For example. the `RunManager` lets you use different run-creation strategies [Add link] and returns a pre-configured `RunService` you can then use.

In most cases you'd work with the Managers directly, but the services are also available if you just want to communicate with the REST APIs.